stability-checker/before-commit.sh

```diff
@@ -78,17 +66,16 @@ fi
 ##
 # GO IMPORTS & FMT
 ##
-go build -o goimports-vendored ./vendor/golang.org/x/tools/cmd/goimports
+go get golang.org/x/tools/cmd/goimports
 buildGoImportResult=$?
 if [ ${buildGoImportResult} != 0 ]; then
 	echo -e "${RED}✗ go build goimports${NC}\n$buildGoImportResult${NC}"
 	exit 1
 fi
 
-goImportsResult=$(echo "${goFilesToCheck}" | xargs -L1 ./goimports-vendored -w -l)
-rm goimports-vendored
+goImportsResult=$(echo "${goFilesToCheck}" | xargs -L1 goimports -w -l)
```

Comment:
I can see in the logfile: https://storage.googleapis.com/kyma-prow-logs/pr-logs/pull/kyma-project_test-infra/2296/pre-master-stability-checker/1254726950681841664/build-log.txt
that if fails with "xargs: goimports: No such file or directory"

Reply:
It is working now. Directory `/home/prow/go/bin/` is defined as GOPATH in prow job and that dir wasn't in PATH so it was needed to explicitly define to use binary from `$GOPATH/bin/`.